content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -177,17 +137,83 @@ different Kubernetes components.
 | `TokenRequestProjection` | `false` | Alpha | 1.11 | 1.11 |
 | `TokenRequestProjection` | `true` | Beta | 1.12 | |
 | `TTLAfterFinished` | `false` | Alpha | 1.12 | |
+| `ValidateProxyRedirects` | `false` | Alpha | 1.10 | 1.13 |
+| `ValidateProxyRedirects` | `true` | Beta | 1.14 | |
 | `VolumePVCDataSource` | `false` | Alpha | 1.15 | |
-| `VolumeScheduling` | `false` | Alpha | 1.9 | 1.9 |
-| `VolumeScheduling` | `true` | Beta | 1.10 | 1.12 |
-| `VolumeScheduling` | `true` | GA | 1.13 | |
-| `VolumeSubpathEnvExpansion` | `false` | Alpha | 1.14 | 1.14 |
-| `VolumeSubpathEnvExpansion` | `true` | Beta | 1.15 | |
 | `VolumeSnapshotDataSource` | `false` | Alpha | 1.12 | - |
-| `ScheduleDaemonSetPods` | `false` | Alpha | 1.11 | 1.11 |
-| `ScheduleDaemonSetPods` | `true` | Beta | 1.12 | |
+| `VolumeSubpathEnvExpansion` | `false` | Alpha | 1.11 | 1.14 |
+| `VolumeSubpathEnvExpansion` | `true` | Beta | 1.15 | |
 | `WatchBookmark` | `false` | Alpha | 1.15 | |
 | `WindowsGMSA` | `false` | Alpha | 1.14 | |
+| `WinDSR` | `false` | Alpha | 1.14 | |
+| `WinOverley` | `false` | Alpha | 1.14 | |
```

Comment:
hello @tengqm . The changes seem fine.
A couple of minor nits:
line 149: spelling: WinOverlay
nit: There may be a few places in the second table where an Alpha feature is listed as defaulting to true.
/lgtm

Reply:
Regarding the spelling error, will fix.
Regarding Alpha feature default to True, I believe it was the true fact. There were some rare cases where the upstream set some Alpha features to true, by mistake, I think.

Reply:
OK